test_unstructured/file_utils/test_model.py

```diff
@@ -10,6 +10,12 @@
 class DescribeFileType:
     """Unit-test suite for `unstructured.file_utils.model.Filetype`."""
 
+    # -- .__lt__() ----------------------------------------------
+
+    def it_is_a_collection_ordered_by_name_and_can_be_sorted(self):
```

Comment:
This is ordered alphabetically, right?

Reply:
Magic methods precede other methods, just like in the class. Is that what you're asking?

Reply:
No, I mean the ordering of FileType on name (below--I commented on the wrong line)

Reply:
Ah, `.extra_name` was out of order. fixed that ;)

Reply:
But yes, the ordering is alphabetical by name, the `.name` attribute of an `enum.Enum` member being the way you access it from the `Enum` class, like the name of `FileType.DOCX` is `"DOCX"`.